openedx/core/djangoapps/course_groups/migrations/0001_initial.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-from __future__ import unicode_literals
+from __future__ import unicode_literals, absolute_import
```

Comment:
Didn't run isort on the migration files?  Not a big deal, but I expected these to be alphabetized.

Reply:
I think python-modernize mistakenly added two lines of `from __future__ import...`, and then I coalesced the lines without alphabetizing them.